e2e-tests/playwright/tests/functional/channels/drafts/drafts_on_deleted_message.spec.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {PreferenceType} from '@mattermost/types/preferences';
+import {Channel} from '@mattermost/types/channels';
+import {UserProfile} from '@mattermost/types/users';
+import {Post} from '@mattermost/types/posts';
+
+import {test, expect} from '@e2e-support/test_fixture';
+import {createRandomPost} from '@e2e-support/server/post';
+import {createRandomChannel} from '@e2e-support/server';
+
+test('MM-T5435_1 Gloabl Drafts link in sidebar should be hidden when another user deleted root post and user removes the deleted post ', async ({
+    pw,
+    pages,
+}) => {
+    const {adminClient, team, adminUser, user, userClient} = await pw.initSetup();
+
+    if (!adminUser) {
+        throw new Error('Failed to create admin user');
+    }
+
+    try {
+        // # Set preferences to disable tooltips for drafts and threads
+        await userClient.savePreferences(user.id, createNoTooltipForDraftsAndThreadsPreferences(user.id));
```

Comment:
Let's move this globally in the [initSetup](https://github.com/mattermost/mattermost/blob/master/e2e-tests/playwright/support/server/init.ts#L56), preferably with the rest of disabling onboarding/tutorials of a user (same as `cy.apiDisableTutorials`).

Reply:
Done ✅